src/global/Types.ts

```diff
@@ -74,7 +74,7 @@ export interface Specimen {
     type: string
 };
 
-export interface SpecimenDigitalMedia  {
+export interface SpecimenDigitalMedia {
```

Comment:
Wondering if we can generate these interface's from the json schema:
https://www.npmjs.com/package/json-schema-to-typescript
That is also what we do in Java, we would then have one schema and generate the types in either java/typescript or python
Not for this PR but maybe create a task for it

Reply:
Interesting thought. I would have to download the JSON schemas in order to use this I think.
https://naturalis.atlassian.net/browse/DD-736?atlOrigin=eyJpIjoiZDNiZThlMjQwZGQ4NGNlMjg3MDQxOTYxOTFmNDEyYmUiLCJwIjoiaiJ9